twoliter/src/common.rs

```diff
@@ -9,15 +9,16 @@ pub(crate) async fn exec_log(cmd: &mut Command) -> Result<()> {
         log::max_level(),
         LevelFilter::Off | LevelFilter::Error | LevelFilter::Warn
     );
-    exec(cmd, quiet).await
+    exec(cmd, quiet).await?;
+    Ok(())
 }
 
 /// Run a `tokio::process::Command` and return a `Result` letting us know whether or not it worked.
 /// `quiet` determines whether or not the command output will be piped to `stdout/stderr`. When
-/// `quiet=true`, no output will be shown.
-pub(crate) async fn exec(cmd: &mut Command, quiet: bool) -> Result<()> {
+/// `quiet=true`, no output will be shown and will be returned instead.
+pub(crate) async fn exec(cmd: &mut Command, quiet: bool) -> Result<String> {
```

Comment:
Rather than sometimes returning an empty string, I'd prefer to return a `Result<Option<String>>`.